h/static/scripts/admin-users.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+function AdminUsersController(element, window_) {
+  this._form = element.querySelector('#js-users-delete-form');
+
+  function confirmFormSubmit() {
+    return window_.confirm("This will permanently delete all the user's data. Are you sure?");
+  }
+
+  if(this._form) {
```

Comment:
That's the command I was looking for, pushing and then waiting for Hound is not the best workflow :)